### PR TITLE
Added markdown MIME Type

### PR DESCRIPTION
**Pull request by n123456789k**
added `.md` as `text/markdown`

### CODE REVIEW

**Comment by n123456789k** [comment]
RFC for MIME Type: https://tools.ietf.org/html/rfc7763

**Comment by colinodell** [comment]
As the maintainer of league/commonmark, I support this addition :wink:

**Comment by OskarStark** [comment]
What about adding `*.markdown`, too?

File: src/Util/MimeType.php [hunk 1]
```diff
@@ -102,6 +102,7 @@ class MimeType
         'txt' => 'text/plain',
         'text' => 'text/plain',
         'log' => 'text/plain',
+        'md' => 'text/markdown',
```
Comment (OskarStark) [inline src/Util/MimeType.php:106]:
```suggestion
        'markdown' => 'text/markdown',
        'md' => 'text/markdown',        
```

**Comment by n123456789k** [comment]
Added `*.markdown` @OskarStark

**Comment by GrahamCampbell** [comment]
NB This is already available in flysystem v2 which uses https://github.com/thephpleague/mime-type-detection/blob/master/src/GeneratedExtensionToMimeTypeMap.php.

**Comment by n123456789k** [comment]
@GrahamCampbell Can this be merged as a fix to the v1 stable rather than waiting on v2 release.

Also, no idea why it's failing since I've added the other `.markdown` 🤷

**Comment by GrahamCampbell** [comment]
> Can this be merged as a fix to the v1 stable rather than waiting on v2 release.

**Comment by GrahamCampbell** [comment]
> Can this be merged as a fix to the v1 stable rather than waiting on v2 release.

I would have thought so.

> Also, no idea why it's failing since I've added the other .markdown 🤷

It's because of the trailing whitespace you added:

image

**Comment by n123456789k** [comment]
@GrahamCampbell Whitespace removed 👍

**Comment by frankdejonge** [comment]
Thanks @narwy 👍

**Comment by frankdejonge** [comment]
Released as 1.0.67

**Pull request merged**